HW7/task7.py

```diff
@@ -0,0 +1,17 @@
+new_dict = {}
+while True:
+    command = input("Enter a command: ")
+    if command == "add":
+        name = input("Enter a name: ")
+        number = input("Enter a number: ")
+        new_dict[name] = number
+    elif command == "stats":
+        print(len(new_dict))
+    elif command == "delete <name>":
```

Comment:
I think only command wanted to be in comparison
```suggestion
    elif command == "delete":
```